app/code/Epam/Webapi/Api/TestServiceInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+declare(strict_types=1);
+
+namespace Epam\Webapi\Api;
+
+use Doctrine\Common\Collections\ArrayCollection;
+
+interface TestServiceInterface
+{
+    /**
+     * Get customers list
+     *
+     * @return mixed
+     */
+    public function listCustomers(): array;
```

Comment:
It is not allowed to use array return type for webapi calls. Please specify EntityInterface[] type instead.

Reply:
In this module I don't have Entity Interface. I will be able to use it from this PR(https://github.com/jumshud/magento2ce/pull/6/) after merging